gufe/components/proteincomponent.py

```diff
@@ -374,7 +374,7 @@ def to_openmm_positions(self) -> omm_unit.Quantity:
 
         return openmm_pos
 
-    def to_pdb_file(self, out_path: Union[Union[str, bytes, PathLike[str], PathLike[bytes]], io.TextIOBase] = None) -> str:
+    def to_pdb_file(self, out_path: Optional[Union[Union[str, bytes, PathLike[str], PathLike[bytes]], io.TextIOBase]] = None) -> str:
```

Comment:
I'm not sure these are correct as optional, from reading the function body I think we actually need something here as there isn't a default filename